code/modules/mob/living/carbon/human/species_types/roguetown/fey/redcap.dm

```diff
@@ -0,0 +1,260 @@
+//		return list(SPAN_SELF)
+//	if(message_language.type == /datum/language/common)
+//		return list(SPAN_SELF)
+	return message_language.spans
+
+/datum/species/fey/redcap/get_skin_list()
+	return list(
+
+		"Misty Hollow"			= SKIN_COLOR_MISTY_HOLLOW,
+		"Ashen Moor"			= SKIN_COLOR_ASHEN_MOOR,
+		"Tunnelclay"			= SKIN_COLOR_TUNNELCLAY,
+		"Weeping Cliffs"		= SKIN_COLOR_WEEPING_CLIFFS,
+		"Deepsand"				= SKIN_COLOR_DEEPSAND,
+		"Duskshroom Corridor"	= SKIN_COLOR_DUSKSHROOM_CORRIDOR
+	)
+
+/datum/species/fey/redcap/get_hairc_list()
+	return sortList(list(
+	"black - oil" = "181a1d",
+	"black - cave" = "201616",
+	"black - rogue" = "2b201b",
+	"black - midnight" = "1d1b2b",
+
+	"blond - pale" = "9d8d6e",
+	"blond - dirty" = "88754f",
+	"blond - drywheat" = "d5ba7b",
+	"blond - strawberry" = "c69b71",
+
+	"white - snow" = "dee9ed",
+	"white - ice" = "f4f4f4",
+
+	"brown - mud" = "362e25",
+	"brown - oats" = "584a3b",
+	"brown - grain" = "58433b",
+	"brown - soil" = "48322a",
+
+	"red - berry" = "48322a",
+	"red - wine" = "82534c",
+	"red - sunset" = "82462b",
+	"red - blood" = "822b2b",
+
+	"green - grass" = "2a482c",
+	"green - swamp" = "3b482a",
+	"green - leaf" = "2f3c2e",
+	"green - moss" = "3b3c2a"
+
+	))
+
+/datum/species/fey/redcap/random_name(gender,unique,lastname)
+
+	var/randname
+	if(unique)
+		if(gender == MALE)
+			for(var/i in 1 to 10)
+				randname = pick( world.file2list("strings/rt/names/fey/redcap/redcapm.txt") )
+				if(!findname(randname))
+					break
+		if(gender == FEMALE)
+			for(var/i in 1 to 10)
+				randname = pick( world.file2list("strings/rt/names/fey/redcap/redcapf.txt") )
+				if(!findname(randname))
+					break
+	else
+		if(gender == MALE)
+			randname = pick( world.file2list("strings/rt/names/fey/redcap/redcapm.txt") )
+		if(gender == FEMALE)
+			randname = pick( world.file2list("strings/rt/names/fey/redcap/redcapf.txt") )
+	return randname
+
+/datum/species/fey/redcap/random_surname()
+	return " [pick(world.file2list("strings/rt/names/fey/redcap/redcaplast.txt"))]"
+
+/datum/species/fey/redcap/after_creation(mob/living/carbon/human/H)
+	..()
+	H.mind?.special_items.Add(/obj/item/clothing/head/roguetown/feycap)
+
+/datum/species/fey/redcap/check_roundstart_eligible()
+	return TRUE
+
+/obj/item/clothing/head/roguetown/afterattack(atom/target, mob/user, proximity)
+	. = ..()
+	if(!HAS_TRAIT(user, TRAIT_FEY_HAT_DIPPER))
+		return
+	if(!proximity || !check_allowed_items(target, target_self = 1))
+		return
+
+	var/turf/bloodspot = get_turf(target)
+	if(!bloodspot)
+		return
+
+	// Search the turf for any blood puddle objects
+	var/obj/effect/decal/cleanable/blood/puddle/bloodpuddle = locate() in bloodspot
+	if(!bloodpuddle)
+		return
+
+	if(sewrepair) //check if headwear is a hat
+		var/mob/living/carbon/human/H = user
+		H.visible_message(
+			span_notice("[H] begins to wipe up \the [bloodpuddle.name] with \the [src]."),
+			span_notice("I wipe up \the [bloodpuddle.name] with my [src]...")
```

Comment:
```suggestion
			span_notice("I wipe up \the [bloodpuddle] with my [src]...")
```
`\the [foo.name]` is invalid. `[foo]` on its own is the same as `\the [foo]` (except for some edge cases with capitalization; at the start of sentences it's `\The [foo]` instead), and the way you avoid the auto-insertion of `\the` is by using `[foo.name]`.